pkg/config/config.go

```diff
@@ -370,6 +370,11 @@ func DefaultConfig() *Config {
 			Enabled:    true,
 			ListenAddr: ":23232",
 			PublicURL:  "http://localhost:23232",
+			CORS: CORSConfig{
+				AllowedHeaders: []string{"Accept", "Accept-Language", "Content-Language", "Origin"},
+				AllowedMethods: []string{"GET", "HEAD", "POST"},
+				AllowedOrigins: []string{""},
```

Comment:
Shouldn't an empty slice have the same meaning?
```suggestion
				AllowedOrigins: []string{},
```

Reply:
What's the behavior of using Soft Serve without a CORS middleware?

Reply:
afaik before #516 added cors middleware all cors/browser requests failed because soft-serve never returned any cors headers